### PR TITLE
Added list with working search bar

### PR DESCRIPTION
**Pull request by KacperWojdak**
Also changed linking onto '/clublist'

### CODE REVIEW

**Comment by AndrzejBudzyn** [comment]
The backend is already working 100%, now  @KacperWojdak has to graphically handle it for us.

File: frontend/src/ClubList.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                            placeholder={t('searchPlaceholder')}
+                            value={searchTerm}
+                            onChange={handleSearchChange}  
+                        />
+                    </div>
+                </div>
+                <div className="overflow-x-auto rounded mb-4">
+                    <div className="flex flex-col items-center">
+                        {filteredClubs.map((club, index) => (
+                        <div key={index} className={`relative bg-custom-sand rounded-xl flex items-center overflow-visible h-20 my-2 w-full hover:bg-custom-olive`}>
```
Comment (PiotrFedak) [inline frontend/src/ClubList.js:62]:
```suggestion
                        <div key={index} className={`relative bg-custom-sand rounded-xl flex items-center overflow-visible h-20 my-3 w-full hover:bg-custom-olive`}>
```

File: frontend/src/ClubList.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                            <Link to={club.url} className="flex items-center w-full h-full">
+                                <img
+                                    src={club.icon.options.iconUrl}
+                                    alt={club.team}
+                                    className="absolute left-16 w-16 h-16 top-1/2 transform -translate-y-1/2"
```
Comment (PiotrFedak) [inline frontend/src/ClubList.js:67]:
```suggestion
                                    className="absolute left-16 w-24 h-24 top-1/2 transform -translate-y-1/2"
```

**Review by PiotrFedak — COMMENTED** [review]
I think that way it will look better 
11f364ccbdada7519dfb84a0f4242f48

**Review by PiotrFedak — APPROVED** [review]
👍

**Pull request merged**